quote-book/src/error.rs

```diff
@@ -20,6 +21,12 @@ pub enum Error {
     /// Quote not found
     QuoteNotFound,
 
+    /// Quote has a spent keyimage
+    QuoteIsStale,
+
+    /// Ledger related error
+    LedgerError(LedgerError),
```

Comment:
Nit: Rename to `Ledger(LedgerError)`, error is implied since this is inside an `Error` enum